### PR TITLE
Socratic 137

### PR DESCRIPTION
**Pull request by btsea**
WIP

### CODE REVIEW

File: _posts/2024-06-27-bitcoin-developers’-socratic-seminar-136.md [hunk 1]
```diff
@@ -0,0 +1,410 @@
...
+layout: post
+type: socratic
+title: "Bitcoin Developers’ Socratic Seminar 137 (at PubKey!)"
+meetup: "https://www.meetup.com/bitdevsnyc/events/301367442/"
+---
+
+## Announcements
+Please join us for our next Socratic Seminar. A special thank you to our sponsors [Chaincode Labs](https://chaincode.com) and [Wolf NYC](https://wolfnyc.com) for food, refreshments and event space.
+
+If you can't make it to the main event please join us at [PUBKEY](https://pubkey.bar/home) around 9:30PM.
```
Comment (pinheadmz) [inline _posts/2024-06-27-bitcoin-developers’-socratic-seminar-136.md:11]: Actually the whole event is at pubkey !

Comment (btsea) [reply _posts/2024-06-27-bitcoin-developers’-socratic-seminar-136.md]: Ah thanks!

File: _posts/2024-06-27-bitcoin-developers’-socratic-seminar-137.md [hunk 2]
```diff
@@ -0,0 +1,218 @@
...
+    - [#30132 Don't wipe indexes again when continuing a prior reindex](https://bitcoincore.reviews/30132)
+
+- Bitcoin Core general developer meetings
+    - [May 30](https://www.erisian.com.au/bitcoin-core-dev/log-2024-05-30.html#l-166)
+    - [June 6](https://www.erisian.com.au/bitcoin-core-dev/log-2024-06-06.html#l-68)
+    - [June 13](https://www.erisian.com.au/bitcoin-core-dev/log-2024-06-13.html#l-81)
+    - [June 20](https://www.erisian.com.au/bitcoin-core-dev/log-2024-06-20.html#l-127)
+
+- Lightning Specification meeting
+    - <!--- TODO replace: [December 5th](https://github.com/lightning/bolts/issues/1046) --->
```
Comment (murchandamus) [inline _posts/2024-06-27-bitcoin-developers’-socratic-seminar-137.md:50]: Empty section

File: _posts/2024-06-27-bitcoin-developers’-socratic-seminar-137.md [hunk 3]
```diff
@@ -0,0 +1,218 @@
+---
+layout: post
+type: socratic
+title: "Bitcoin Developers’ Socratic Seminar 137 (at PubKey!)"
+meetup: "https://www.meetup.com/bitdevsnyc/events/301367442/"
+---
+
+## Announcements
+Please join us for our next Socratic Seminar, hosted at PubKey! A special thank you to our sponsors [Chaincode Labs](https://chaincode.com) and [Wolf NYC](https://wolfnyc.com) for food, refreshments and event space.
```
Comment (murchandamus) [inline _posts/2024-06-27-bitcoin-developers’-socratic-seminar-137.md:9]: Pubkey spells itself Pubkey. Also, updating the sponsor notes:

```suggestion
Please join us for our next Socratic Seminar, hosted at Pubkey! Food and drinks will be available for purchase from Pubkey. A limited tab will be available on a first-come basis. We thank [Wolf NYC](https://wolfnyc.com) for sponsoring the event.
```

**Comment by murchandamus** [comment]
Opened PR in #163

**Pull request merged**